DataProc/TPCSPCalibration/procTBinMap.sh

```diff
@@ -69,7 +69,7 @@ extractEnvVars()
 [[ $# -lt 3 ]] &&  Usage && exit 1
 echo "Arguments: 1 = $1, 2 = $2, 3 = $3"
 
-source $ALICE_PHYSICS/PWGPP/scripts/alilog4bash.sh  
+[[ -e $ALICE_PHYSICS/PWGPP/scripts/alilog4bash.sh ]] && source "$ALICE_PHYSICS"/PWGPP/scripts/alilog4bash.sh || source "$ALICE_ROOT"/libexec/alilog4bash.sh
```

Comment:
This fixes a problem never noticed on the Grid: that file was moved and TPC SP jobs haven't had any log output since a while.